lib/broccoli/glimmer-app.ts

```diff
@@ -417,27 +415,19 @@ Please run the following to resolve this warning:
   }
 
   private cssTree() {
-    // should really make SASS support to be opt-in, so that
-    // we can properly honor the `GlimmerAppOptions.trees.src`
-    // abstraction here, but for now we still require `src` to be a
-    // "real" path on disk that we check
     let stylesPath = path.join(resolveLocal(this.project.root, 'src'), 'ui', 'styles');
 
     if (fs.existsSync(stylesPath)) {
-      // Compile SASS if app.scss is present
-      // (this works with imports from app.scss)
-      let scssPath = path.join(stylesPath, 'app.scss');
-      if (fs.existsSync(scssPath)) {
-        return compileSass([stylesPath], 'app.scss', this.outputPaths.app.css, {
-          annotation: 'Funnel: scss'
-        });
-      }
-
-      // Otherwise concat all the css in the styles dir
-      return concat(new Funnel(stylesPath, {
-        include: ['**/*.css'],
-        annotation: 'Funnel: css'}),
-        { outputFile: this.outputPaths.app.css });
+      let cssTree = addonProcessTree(this.project, 'preprocessTree', 'css',
+        new Funnel(stylesPath, {
+          destDir: '/src/ui/styles',
+          include: ['**/*.css', '**/*.scss'],
```

Comment:
I don't think we should need to filter here at all.  We should just pass through `src/ui/styles` without filtering.  Filtering breaks preprocessors that also access/handle non-css assets.  One example being [eyeglass](https://github.com/sass-eyeglass/eyeglass), which has the ability to ensure asset urls are correct when distributed as node modules.